Source/Initialization/WarpXInitData.cpp

```diff
@@ -876,10 +876,16 @@ WarpX::InitializeExternalFieldsOnGridUsingParser (
        std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& edge_lengths,
        std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& face_areas,
        const char field,
-       const int lev)
+       const int lev, PatchType patch_type)
 {
 
-    const auto dx_lev = geom[lev].CellSizeArray();
+    auto dx_lev = geom[lev].CellSizeArray();
+    amrex::IntVect refratio = ( (lev > 0 ) ? WarpX::RefRatio(lev-1) : amrex::IntVect(1) );
```

Comment:
Operator precedence in C++ makes the `()` superfluous :)
```suggestion
    amrex::IntVect refratio = (lev > 0 ) ? WarpX::RefRatio(lev-1) : amrex::IntVect(1);
```
https://en.cppreference.com/w/cpp/language/operator_precedence